tests/unit/EntityMenu.spec.ts

```diff
@@ -503,6 +503,9 @@ describe.skip('Entity Menu - Request AGM Extension click tests', () => {
     vm = wrapper.vm
     vi.spyOn(vm, 'enableAgmExtension', 'get').mockReturnValue(true)
 
+    // Fix TypeError: GetFeatureFlag(...).includes is not a function test error (unrelated to extensions)
+    vi.spyOn(vm, 'enableAgmLocationChg', 'get').mockReturnValue(true)
```

Comment:
Oh, so this is because the mock for GetFeatureFlag (lines 486-487) returns a boolean instead of an array.

I'd prefer to see a smarter mock above than this "unrelated" work-around, if possible.

Reply:
The issue is inEntityMenu.vue on line 260-261:
```javascript
  get enableAgmLocationChg (): boolean {
    return !!GetFeatureFlag('supported-agm-location-chg-entities').includes(this.getLegalType)
  }
```

The tests are actually passing in here. The compiler is showing however an error that we cannot use includes on something that hasn't been set. Keep in mind this is for location change which is why it's unrelated since this test is for extensions.

The current code (latest commit) mocks the return value of the getter above to be `BEN`.

Please let me know if you that's not OK Sev.

Reply:
See my other comment below. Resolving this one.